diff --git a/node_modules/react-native-vector-icons/.DS_Store b/node_modules/react-native-vector-icons/.DS_Store
new file mode 100644
index 0000000..b7fa5d4
Binary files /dev/null and b/node_modules/react-native-vector-icons/.DS_Store differ
diff --git a/node_modules/react-native-vector-icons/android/.DS_Store b/node_modules/react-native-vector-icons/android/.DS_Store
new file mode 100644
index 0000000..5008ddf
Binary files /dev/null and b/node_modules/react-native-vector-icons/android/.DS_Store differ
diff --git a/node_modules/react-native-vector-icons/android/build.gradle b/node_modules/react-native-vector-icons/android/build.gradle
index 28e2fcc..ffbfd8e 100755
--- a/node_modules/react-native-vector-icons/android/build.gradle
+++ b/node_modules/react-native-vector-icons/android/build.gradle
@@ -11,12 +11,12 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-  compileSdkVersion 26
-  buildToolsVersion "26.0.1"
+  compileSdkVersion 28
+  buildToolsVersion "28.0.3"
 
   defaultConfig {
     minSdkVersion 16
-    targetSdkVersion 26
+    targetSdkVersion 28
     versionCode 1
     versionName "1.0"
   }
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-  compile "com.facebook.react:react-native:+"
+  implementation "com.facebook.react:react-native:+"
 }
